Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportColumns.cs

```diff
@@ -16,6 +16,7 @@ public class GDPRExportReportColumns
         public static CsvColumnIdentity DataProcessorControl = new CsvColumnIdentity(nameof(DataProcessorControl), "Tilsyn/kontrol af databehandleren");
         public static CsvColumnIdentity LinkToDirectory = new CsvColumnIdentity(nameof(LinkToDirectory), "Link til fortegnelse");
         public static CsvColumnIdentity RiskAssessment = new CsvColumnIdentity(nameof(RiskAssessment), "Foretaget risikovurdering");
+        public static CsvColumnIdentity RiskAssessmentDate = new CsvColumnIdentity(nameof(RiskAssessmentDate), "RiskAssessmentDate <- change the text");
```

Comment:
change text

Reply:
why not use the one from the UI? 
![image](https://user-images.githubusercontent.com/13116889/206158040-8c961822-8918-4440-92fe-8d625d3b0050.png)